src/profile/components/profileInformation/ProfileInformationAccountManagementLink.tsx

```diff
@@ -32,13 +33,13 @@ function ProfileInformationAccountManagementLink(): React.ReactElement | null {
         />
       </div>
       <div className={styles['link']}>
-        <a
-          href={authenticationMethodReferenceUrl}
-          target="_blank"
-          rel="noopener noreferrer"
+        <NewWindowLink
+          link={authenticationMethodReferenceUrl}
+          title={t('profileInformation.doGoToAccountManagement')}
+          hideIcon
```

Comment:
I think this link would be more understandable to those with proper vision, if the icon _is_ shown.

Reply:
There's the ">" icon from already, that's why the second one is hidden. The "new window"-icon does not fit the layout.

Reply:
Does the ">" icon have a common meaning? I haven't figured one out. To me it indicates that it could be an accordion, or a menu or whatever. _Instead of_ the ">" icon, the "arrow out of box" icon would more clearly indicate and external link.

Reply:
In the design, the ">" emphazised the text "Go to..." That's how I understand it. And the text+icon is aligned to the right because the ">" points that direction. There would be no need for that alignment, if there was only "Jump + box" :)

Reply:
Have any designs actually used this "arrow out of box" icon for external links? Consistency is also important in UX and I guess it would be good to use only one solution to indicate any given thing.

But yeah, we're going perhaps a bit too much on the territory of UI design here, where the topic was actually to improve accessibility. We can leave the ">" icon be as it is.